milestone-3/frontend/src/components/recipePreview.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+
+interface Recipe {
+    name: string;
+    image: string;
+    desc: string;
+}
+interface RecipePreviewProps extends Recipe {
+    external?: boolean;
+  }
+
+
+function RecipePreview(props: RecipePreviewProps) {
+    return (
+    <div className="sub-section">
+    <h2>
+        <Link to={props.external ? `/externalRecipe/${props.name}` : `/recipe/${props.name}`} className="text">{props.name}</Link>
+    </h2>
+    <div className="sub-section-content">
+        <img className="recipe-images-home" src={props.image} alt="picture of banana bread" width="200" height="200"/>
```

Comment:
your alt text here is not unique to the recipe props data passed in, so it will always show banana bread even if you're on a different page. It's good to change this to something passed down about the recipe, like the recipe name or description.